Sources/TuistCore/MetadataProviders/XCFrameworkSignatureProvider.swift

```diff
@@ -0,0 +1,99 @@
+import Command
+import CryptoKit
+import FileSystem
+import Foundation
+import Path
+import TSCBasic
+import TuistSupport
+import XcodeGraph
+
+enum XCFrameworkSignatureProviderError: LocalizedError, Equatable {
+    case codesignOutputMissing
+    case certificateFileReadFailed
+    case appleSignedXCFrameworkMissingDetails(teamIdentifier: String?, teamName: String?)
+
+    var errorDescription: String {
+        switch self {
+        case .codesignOutputMissing:
+            return "codesign finished, but no output file was found."
+        case .certificateFileReadFailed:
+            return "Failed to read certificate file."
```

Comment:
Whose certificate? Let's include the path to the XCFramework or the certificate here.